google-login-plugin/testSrc/com/google/gct/login/GoogleLoginPrefsTest.java

```diff
@@ -17,40 +17,16 @@
 
 import junit.framework.TestCase;
 
-import java.io.*;
-
 
 /**
- * Tests for {@link GoogleLoginPrefs}
+ * Tests for {@link GoogleLoginPrefs}.
  */
+public class GoogleLoginPrefsTest extends TestCase {
 
-//public class GoogleLoginPrefsTest extends TestCase {
-public class GoogleLoginPrefsTest {
-  // The required permission for the preference file
-  private static final String PREFERENCE_FILE_PERMISSION = "-rw-r-----";
-
-  /**
-   * Tests that the Google Login credentials file is only accessible by the user.
-   */
-  // TODO: when upgraded to Java7, use java.nio.file.Files.getPosixFilePermissions
-  public void testPreferenceFileAccessibility() throws IOException {
-    String preferencesPath = GoogleLoginPrefs.getPreferencesPath();
-    File userRootDir = new File(System.getProperty("java.util.prefs.userRoot",
-      System.getProperty("user.home")), ".java/.userPrefs");
-    String absolutePrefPath = userRootDir.getAbsolutePath() + preferencesPath + "/prefs.xml";
-
-    // Get the preference file's permissions
-    Process process = Runtime.getRuntime().exec("ls -al " + absolutePrefPath);
-
-    // Parse the output from the process
-    InputStream inputStream = process.getInputStream();
-    BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream));
-    String line = reader.readLine();
-    inputStream.close();
-    String[] splitOutput = line.split(" ");
-
-    // Check the output
-    //assertTrue(splitOutput.length > 1);
-    //assertEquals(PREFERENCE_FILE_PERMISSION, splitOutput[0]);
+  public void testActiveUser() {
+    GoogleLoginPrefs.saveActiveUser("cheese");
```

Comment:
Would this work:

PREFERENCES_PATH = ApplicationManager.getApplication().isUnitTestMode() ? "/com/google/gct/login" : "/com/google/gct/login/test";

and we can delete test directory after.